app/views/foreman/job_templates/install_errata_-_katello_ansible_default.erb

```diff
@@ -15,12 +15,10 @@ kind: job_template
 %>
 
 <% if @host.operatingsystem.family == 'Suse' -%>
-<% advisories = input(:errata).split(',').join(' ') -%>
-<%= render_template('Run Command - Ansible Default', :command => "zypper -n install -t patch #{advisories}") -%>
-<% elsif @host.operatingsystem.family == 'Debian' -%>
-<% advisories = input(:errata).split(',').map { |e| @host.debs_for_erratum(e) }.join(' ') -%>
-<%= render_template('Run Command - Ansible Default', :command => "apt-get -o Dpkg::Options::=\"--force-confdef\" -o Dpkg::Options::=\"--force-confold\" -y --only-upgrade install -y #{advisories}") %>
+  <% advisories = input(:errata).split(',').join(' ') -%>
+  <% render_error(N_('No errata found with the given name')) if advisories.blank? -%>
```

Comment:
For the "errata by search query" template below, the error message is correct.

But for these other two templates, this is just after manipulating the input string; we haven't yet searched for any errata.
```suggestion
  <% render_error(N_('At least one errata ID is required')) if advisories.blank? -%>
```

Reply:
update: If I send a blank input, it doesn't even get to the point where it would render the template error. Instead, I just see this in the log:
```
Backtrace for 'Action failed' error (ActiveRecord::RecordNotSaved): Validation failed: Template install errata - katello script default: Input errata: Value can't be blank
```

So I'm not sure these two errors are necessary in the templates.